app/assets/javascripts/controllers/container_topology/container_topology_controller.js

```diff
@@ -179,17 +163,17 @@ angular.module('topologyApp', ['kubernetesUI', 'ui.bootstrap'])
         entity_url = "ems_container";
         break;
       case "Container":
-        entity_url = class_name(d);
+        entity_url = d.item.kind.toLowerCase();
         action = '/explorer';
       default :
-        entity_url = class_name(d);
+        entity_url = _.snakeCase(d.item.kind);
```

Comment:
> Uh, what kinds will go through this case? Seems like this will change the output of dblclick for Pod from container_group to pod.

the d.item.kind has changed from pod to containergroup in #5930, and the display_kind is now pod, so this change is aligning with that.